addons/phone_validation/tools/phone_validation.py

```diff
@@ -46,7 +46,7 @@ def phone_parse(number, country_code):
                 else:
                     raise UserError(_('Impossible number %s: too many digits.', number))
             else:
-                raise UserError(_('Impossible number %s: probably invalid number of digits.', number))
+                raise UserError(_("The format of the number %s is incorrect unless you're dialing aliens! Let's fix it.", number))
```

Comment:
Replace by "The phone number %s is incorrect! Let's fix it - you are not dialing aliens."